pkgs/tools/text/ansifilter/default.nix

```
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ boost lua ];
 
+  postPatch = if !stdenv.isDarwin then "" else ''
+    substituteInPlace src/makefile --replace CC=g++ CC=clang++
+  '';
```

This can be made more generic with:

```suggestion
  postPatch = ''
    substituteInPlace src/makefile --replace "CC=g++" "CC=c++"
  '';
```

This has the added benefit that it would also work on Linux with another C++ compiler (e.g. `clang++`).

---

I could have just done the new "commit suggestion" feature, but instead I just force-pushed the exact same change to the branch.

@danieldk I tested this on both my Linux and Mac laptops.  It works fine.  Can you re-review this (and hopefully approve it)?

Thanks.